apidoc/Titanium/Geolocation/Geolocation.yml

```diff
@@ -330,18 +324,18 @@ events:
   - name: calibration
     summary: Fired when the device detects interface and requires calibration.
     description: |
-        When this event is fired, the OS calibration UI is being displayed to the end user.
+        When this event is fired, the iOS calibration UI is being displayed to the end user.
     platforms: [iphone, ipad]
 
   - name: heading
     summary: Fired when an heading update is received.
     properties:
       - name: code
-        summary: If `success` is false, the error code if available.
+        summary: If `success` is `false`, the error code is available.
         platforms: [android, iphone, ipad]
 
       - name: success
-        summary: Indicate if the heading event was successfully received.
+        summary: Indicate if the heading event was successfully received. Android returns this since SDK 7.4.0.
```

Comment:
Should this change be included since it hasn't been released yet (SDK 7.3.0 is pending and 7.4.0 is the next release).

Reply:
This is for our master branch.